mock-pop-mining/src/test/java/org/veriblock/sdk/mock/VeriBlockBlockDataTest.java

```diff
@@ -57,19 +73,117 @@ private void checkBlockData(VeriBlockBlockData blockData) {
         }
     }
 
-    private VeriBlockBlockData createBlockData(Random random, int regularCount, int popCount, int maxSize) {
+    private byte[] randomBytes(Random random, int count) {
+        byte[] data = new byte[count];
+        random.nextBytes(data);
+        return data;
+    }
+
+    private Sha256Hash randomSha256Hash(Random random) {
+        return Sha256Hash.wrap(randomBytes(random, Sha256Hash.BITCOIN_LENGTH));
+    }
+
+    private Address randomAddress(Random random) {
```

Comment:
this looks like a good utility function, maybe you can create a class, which is created with `Random` instance?

Reply:
this is a good idea. probably can do it as part of the test refactoring task that is in the todo